src/bots/mascot-bot.js

```diff
@@ -59,6 +80,51 @@ class MascotBot extends SlackBot {
     }
   }
 
+  postMessage(...args) {
+    if (this.debug) {
+      this.log('API limited in debug mode', true);
```

Comment:
I wish this was more helpful, like "postMessage is not allowed in debug mode. Use postTo instead."